Gopkg.lock

```diff
@@ -142,17 +142,62 @@
   version = "v0.0.5"
 
 [[projects]]
-  digest = "1:0496f0e99014b7fd0a560c539f51d0882731137b85494142f47e550e4657176a"
```

Comment:
is there something specific required by upgrading the lock file?

Reply:
this digest changed because i use the testify/mock sub package for my tests